### ISSUE TITLE
EOL conversion should be ignored when comparing two versions

### ISSUE THREAD

**Issue by spoji** [mozilla/addons-code-manager#1475]
This compare url https://code.addons.mozilla.org/en-US/compare/920304/versions/4957090...4976235/ shows that all the codebase has been changed. This is because the dev uploaded a new version with converted EOL.

### What happened?
One example of this can be found at https://code.addons.mozilla.org/en-US/compare/920304/versions/4957090...4976235/?path=app%2Feventpages%2Fblock.html

In this file, only lines number 24 and 25 actually have real differences. The rest is because of "0D 0A" to "0A" EOL convertions.

![image](https://user-images.githubusercontent.com/2171145/76017158-1ec8f280-5eec-11ea-8d5a-727971be0e03.png)

![image](https://user-images.githubusercontent.com/2171145/76017121-0b1d8c00-5eec-11ea-9c66-1f21377faad9.png)

### What did you expect to happen?
To have a result similar to the old review tool : https://reviewers.addons.mozilla.org/en-US/firefox/files/compare/3520605...3501462/file/app/eventpages/block.html#top

**Comment by diox** [comment]
I don't have access to the old file viewer in prod, so to be clear: the file does show up as modified in the old file viewer sidebar, but the diff is empty for this file because it's only EOL changes, right ?

**Comment by spoji** [comment]
Yes, the file does show up as modified in the old file viewer sidebar. The diff is almost empty beside the two real line changed (lines 24, 25).

![old_tool](https://user-images.githubusercontent.com/2171145/76087556-67cc8580-5f84-11ea-832c-3413961d8ff0.png)

On a side note, while the actual code diff works fine for this use case in the old system, the sidebar has the same issue has the new system: it shows that all files have been modified. When you click on one of them, you are presented however with a proper code diff result.

**Comment by bobsilverberg** [comment]
This sounds like something we should try to fix, but it also sounds like the fix would be best on the server, where the diff and change info is generated, as opposed to on the client. Do you think we should transfer this to addons-server, @diox?
